common/txmgr/txmgr.go

```diff
@@ -508,7 +507,7 @@ func (b *Txm[CHAIN_ID, HEAD, ADDR, TX_HASH, BLOCK_HASH, R, SEQ, FEE]) SendNative
 		FeeLimit:       gasLimit,
 		Strategy:       NewSendEveryStrategy(),
 	}
-	etx, err = b.txStore.CreateTransaction(txRequest, chainID)
+	etx, err = b.txStore.CreateTransaction(context.TODO(), txRequest, chainID)
```

Comment:
We should probably wire this one up too, no? I think it actually comes from an external API call upstream, so it should be easy.